Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -152,7 +160,7 @@ const StatusBar = React.createClass({
     animated: React.PropTypes.bool,
     /**
      * The background color of the status bar.
-     * @platform android
+     * @platform android windows
```

Comment:
Should I add this or not? I wonder what is the correct syntax 
`@platform android windows` or `@platform android, windows `